services/submission/src/main/resources/application.yaml

```diff
@@ -23,6 +23,8 @@ services:
     random-key-padding-multiplier: ${RANDOM_KEY_PADDING_MULTIPLIER:1}
     # The ApacheHttpClient's connection pool size.
     connection-pool-size: 200
+    # The maximum request size accepted by the SubmissionController in Bytes.
+    maximum-request-size: 102400
```

Comment:
According to @stevesap 's comment, I would ask to consider having this an environment variable with default. Please see e.g. `random-key-paddin-multiplier`.